example/01/main_test.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+   // "testing"
+	"math/rand"
+	"github.com/axamon/gobrain"
```

Comment:
Shouldn't it be `github.com/goml/gobrain`?

Reply:
yea fixed it